tests/components/application-connector/test/application-gateway/runner_test.go

```diff
@@ -0,0 +1,37 @@
+package application_gateway
+
+import (
+	"context"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var applications = []string{"basic-cases", "path-cases", "k8s-test"}
+
+func (gs *GatewaySuite) TestCases() {
```

Comment:
I would change this to be more descriptive as the name of this function will be printed on the go test report.
Maybe something like `TestGetRequest`? Seems a bit general but it describes what is being tested: Get requests to Application Gateway with various settings.